code/style.css

```diff
@@ -1,4 +1,189 @@
-html {
-  background: #0025ff;
-  color: #fff;
+* {
+  box-sizing: border-box; /* Doing like this, is it valid for the whole page and elements? */
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 18px;
+  color: #056676;
+  background: #5eaaa8; /* fallback for old browsers */
+  background: -webkit-linear-gradient(
+    to right,
+    #a3d2ca,
+    #5eaaa8
+  ); /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(
+    to right,
+    #a3d2ca,
+    #5eaaa8
+  ); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+}
+
+/*** HEADER ***/
+header {
+  background-color: #e8ded2;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 20px 0px;
+}
+
+.site-logo-name {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  margin-left: 50px;
+  font-size: 24px;
+  font-weight: bold;
+}
+
+.logo {
+  width: 100px;
+  height: 100px;
+}
+
+nav {
+  display: none;
+}
+
+nav a:hover {
+  font-size: 24px;
+  font-weight: bold;
+  border-top: 2px solid #056676;
+  border-bottom: 2px solid #056676;
+}
```

Comment:
This hover styling makes all the links jump around a bit. Consider giving them a fixed width or wrap them in a div with a fixed width so that you always know how much space they will take up (to avoid the jumping around)